tests/components/icloud/test_config_flow.py

```diff
@@ -275,7 +284,7 @@ async def test_login_failed(hass: HomeAssistantType):
             data={CONF_USERNAME: USERNAME, CONF_PASSWORD: PASSWORD},
         )
         assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
-        assert result["errors"] == {CONF_USERNAME: "invalid_auth"}
+        assert result["errors"] == {CONF_PASSWORD: "invalid_auth"}
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6309587)